### DISCUSSION TITLE
Add a ‘v2alpha-{arch}-latest’ image tag to container registry

### DISCUSSION THREAD

**Discussion by binaryfire** [zitadel/zitadel#4038] [Ideas]
Hi guys!

We’re keen to start testing v2 alpha with our project but we’d like to automate updates. We use Docker Swarm and are deploying via a compose file.

Would it be possible to add static image tags for the latest build? Eg. a ‘v2alpha-amd64-latest’ tag. Then we could pull this each night to update to the newest build. 

Our current update process involves manually checking the package list in GitHub and updating the version number in the compose file, which isn’t ideal. I could automate this with a script but a static tag would be a much cleaner solution

Cheers

**Comment by fforootd** [comment]
Hi @binaryfire 

Yeah we have this on the radar 😊 and we definitely will soon create a tag that represents a release channel.

@thesephirot is looking mg into this
